android/src/main/java/com/urbanairship/reactnative/UrbanAirshipReactModule.java

```diff
@@ -98,6 +99,17 @@ public void onHostPause() {
             public void onHostDestroy() {
 
             }
+
+            public void checkOptIn(Context context) {
+                boolean optIn = UAirship.shared().getPushManager().isOptIn();
```

Comment:
Not unless we want the preferences class to do this which would be a side effect